enterprise/management/commands/backfill_learner_role_assignments.py

```diff
@@ -65,17 +64,11 @@ def backfill_learner_role_assignments(self, options):
 
                 user = User.objects.get(id=ecu.user_id)
                 enterprise_customer = ecu.enterprise_customer
-
-                try:
-                    role_assignment, created = SystemWideEnterpriseUserRoleAssignment.objects.get_or_create(
-                        user=user,
-                        role=role,
-                        enterprise_customer=enterprise_customer
-                    )
-                except MultipleObjectsReturned:
-                    log.info('Found MultipleObjectsReturned for user %s. Skipping.', user)
-                    continue
-
```

Comment:
This piece of code is not required anymore now that we are enforcing a unique constraint.